src/settings/settingsdialog.cpp

```diff
@@ -91,6 +91,10 @@ SettingsDialog::SettingsDialog(MainWindow *parent)
 
     ui->ocrLanguagesListWidget->addLanguages(parent->ocr()->availableLanguages());
 
+    ui->proxyTypeComboBox->addItem(tr("Use system settings"), QNetworkProxy::NoProxy);
+    ui->proxyTypeComboBox->addItem(QStringLiteral("SOCKS5"), QNetworkProxy::Socks5Proxy);
+    ui->proxyTypeComboBox->addItem(QStringLiteral("HTTP"), QNetworkProxy::HttpProxy);
```

Comment:
Host/Port fields are disabled now if HTTP type is used.

Reply:
Thanks, fixed!